chromadb/test/conftest.py

```diff
@@ -30,16 +30,80 @@
 from chromadb.types import SeqId, OperationRecord
 from chromadb.api.client import Client as ClientCreator
 
+VALID_PRESETS = ["fast", "normal", "slow"]
+CURRENT_PRESET = os.getenv("PROPERTY_TESTING_PRESET", "normal")
```

Comment:
I don't love the mix in terminology between profile/preset, but `TEST_PROFILE` doesn't have the same connotations, and `HYPOTHESIS_PROFILE` isn't great if we want to share the same env var between multiple languages